src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -70,7 +70,7 @@ import {toCachedTx, TransactionCache} from './shelley/transactionCache'
 import {yoroiSignedTx} from './signedTx'
 import {NetworkId, WalletImplementationId, WalletInterface, YoroiProvider} from './types'
 import {yoroiUnsignedTx} from './unsignedTx'
-import {generateUtxoStorage} from './utxoStorage'
+import {makeUtxoStorage} from './utxoStorage'
```

Comment:
just normalising the names https://github.com/Emurgo/yoroi-mobile/pull/2209/files#diff-35be0faa23c723910b92f1d9db6b12ae6568381be0ba00ac07d2c5d3a08c921cR94